comparisons/CompareGraphs.py

```diff
@@ -83,7 +92,13 @@
         hToCompare[iFile].SetStats(0)
     else:
         hToCompare[iFile].SetName(f'g{iFile}')
-    SetObjectStyle(hToCompare[iFile], color=GetROOTColor(color), markerstyle=GetROOTMarker(marker), fillstyle=0)
+    SetObjectStyle(hToCompare[iFile],
+                   color=GetROOTColor(color),
+                   markerstyle=GetROOTMarker(marker),
+                   markersize=markersize,
+                   linewidth=linewidth,
+                   fillstyle=fillstyle[iFile],
+                   fillalpha=fillalpha[iFile])
```

Comment:
Remove `[iFile]`

Reply:
Hi @fcatalan92! Thank you for your comment. Actually, the idea behind the 'iFile' was to use different styles and transparencies for different Files. Indeed, when I wrote the code, I had to represent overlapping points with different error bars, hence it was useful to have this feature. However, I can change it.

Reply:
But now you are already iterating over the lists in the for loop, so `fillstyle` and `fillalpha` are not list anymore, right?

Reply:
Yes, you're right. Sorry, I fix it.